tests/cypress/e2e/features/annotations_actions.js

```diff
@@ -0,0 +1,292 @@
+            cy.get('.cvat-action-runner-disable-autosave-recommendation').should('not.exist');
+            cy.closeAnnotationsActionsModal();
+
+            cy.openSettings();
+            cy.contains('Workspace').click();
+            cy.get('.cvat-workspace-settings-auto-save').within(() => {
+                cy.get('[type="checkbox"]').should('not.be.checked');
+            });
+            cy.closeSettings();
+        });
+    });
+
+    describe('Test action: "Remove filtered shapes"', () => {
+        const ACTION_NAME = 'Remove filtered shapes';
+
+        before(() => {
+            const shapes = frames.map((frame) => [{
+                type: 'rectangle',
+                occluded: false,
+                outside: false,
+                z_order: 0,
+                points: [250, 350, 350, 450],
+                rotation: 0,
+                attributes: [],
+                elements: [],
+                frame,
+                label_id: labels[0].id,
+                group: 0,
+                source: 'manual',
+            }, {
+                type: 'rectangle',
+                occluded: false,
+                outside: false,
+                z_order: 0,
+                points: [350, 450, 450, 550],
+                rotation: 0,
+                attributes: [],
+                elements: [],
+                frame,
+                label_id: labels[1].id,
+                group: 0,
+                source: 'manual',
+            }]).flat();
+
+            const tracks = [{
+                label_id: labels[0].id,
+                frame: 0,
+                group: 0,
+                source: 'manual',
+                attributes: [],
+                elements: [],
+                shapes: [{
+                    type: 'rectangle',
+                    occluded: false,
+                    z_order: 0,
+                    rotation: 0,
+                    outside: false,
+                    attributes: [],
+                    frame: 0,
+                    points: [250, 350, 350, 450],
+                }],
+            }];
+
+            const tags = frames.map((frame) => [{
+                frame,
+                label_id: labels[0].id,
+                source: 'manual',
+                group: 0,
+                attributes: [],
+            }, {
+                frame,
+                label_id: labels[1].id,
+                source: 'manual',
+                group: 0,
+                attributes: [],
+            }]).flat();
+
+            cy.window().then((window) => {
+                window.cvat.server.request(`/api/jobs/${jobID}/annotations`, {
+                    method: 'PUT',
+                    data: { shapes, tracks, tags },
+                });
+            });
+        });
+
+        beforeEach(() => {
+            cy.visit(`/tasks/${taskID}/jobs/${jobID}`);
+            cy.get('.cvat-canvas-container').should('not.exist');
```

Comment:
Do we really need this check?

Reply:
Yes, otherwise immediately after `cy.visit` page did not updated yet and the next check `cy.get('.cvat-canvas-container').should('exist').and('be.visible');` returns `true`.  At the same moment it starts executing tests body, but the page starts reloading

Reply:
Then maybe just check for 
```
            cy.get('.cvat-spinner').should('not.exist');
```
Instead of double-checking the canvas-container

Reply:
Looks like the suggestion is not working